### PR TITLE
Rewrite raw queries

### PR DESCRIPTION
**Pull request by thibaultmeyer**
- Use real .Count() rather than SELECT + len(array)
- Raw "SELECT" queries now use XORM
- Clean other XORM queries (now we can get the real line where an issue appear)
- `gofmt` on all model files

### CODE REVIEW

File: models/org.go [hunk 1]
```diff
@@ -506,32 +536,29 @@ func (org *User) GetUserRepositories(userID int64, page, pageSize int) ([]*Repos
...
-	GROUP BY repository.id
-	ORDER BY updated_unix DESC`,
-		strings.Join(base.Int64sToStrings(teamIDs), ",")),
-		org.ID, false)
+	repoCount, err := x.
+		Join("INNER", "team_repo", "`team_repo`.repo_id=`repository`.id").
+		Where("(`repository`.owner_id=? AND `repository`.is_private=?)", org.ID, false).
+		Or("team_repo.team_id IN (?)", strings.Join(base.Int64sToStrings(teamIDs), ",")).
+		GroupBy("`repository`.id").
+		Count(&Repository{})
```
Comment (metalmatze) [inline models/org.go:556]: :+1:

File: models/release.go [hunk 2]
```diff
@@ -127,7 +127,9 @@ func GetRelease(repoID int64, tagName string) (*Release, error) {
 // GetReleaseByID returns release with given ID.
 func GetReleaseByID(id int64) (*Release, error) {
 	rel := new(Release)
-	has, err := x.Id(id).Get(rel)
+	has, err := x.
```
Comment (metalmatze) [inline models/release.go:130]: I think this could have be left on line, it's not that long. But it's nitpicking.

**Comment by metalmatze** [comment]
Overall _really_ awesome changes! 

My only comment is one may lines that could have been left one-liners.

There's really no need to write

``` go
    err = x.
        Where("org_id=?", orgID).
        And("is_active=?", true).
        Find(&ws)
```

instead of:

``` go
    err = x.Where("org_id=?", orgID).And("is_active=?", true).Find(&ws)
```

It's not that much.

LGTM

**Comment by andreynering** [comment]
> My only comment is one may lines that could have been left one-liners.

Somethings fall to personal preference. I prefer it in multiple lines, but I'm OK with any style.

I'd also prefer some more spaces inside SQLs:

``` go
x.Where("org_id = ?", orgID)
// instead of
x.Where("org_id=?", orgID)
```

**Comment by andreynering** [comment]
LGTM

**Pull request merged**